domain/secretbackend/service/service_test.go

```diff
@@ -1005,6 +1006,86 @@ func (s *serviceSuite) TestWatchSecretBackendRotationChanges(c *gc.C) {
 	wC.AssertNoChange()
 }
 
+func (s *serviceSuite) TestGetModelSecretBackend(c *gc.C) {
+	defer s.setupMocks(c).Finish()
+	svc := newService(
+		s.mockState, s.logger, s.clock, nil,
+	)
+
+	modelUUID := modeltesting.GenModelUUID(c)
+	gomock.InOrder(
```

Comment:
Please remove the In Order and right expect without an expectation on ordering.

Reply:
Nah, it has to be in order for the test. Just didn't want to split to this simple test to too many small tests.